README.md

```diff
@@ -1 +1,83 @@
-# CytoPipe
\ No newline at end of file
+# Table of contents
+
+- [Table of contents](#table-of-contents)
+- [CytoPipe](#cytopipe)
+- [Formatting](#formatting)
+  - [smkfmt](#smkfmt)
+- [Testing](#testing)
+  - [Dry Runs](#dry-runs)
+
+# CytoPipe
+
+# Formatting
+
+## smkfmt
+
+`snakefmt` is a `snakemake` code formatter that is developed and maintained by the developers of `snakemake`.
+
+In order to use `snakefmt`, `snakemake` must be installed, which can be found here: [snakemake-installation](https://snakemake.readthedocs.io/en/stable/getting_started/installation.html)
+
+`snakefmt` installation processes can be found in the [snakefmt](https://github.com/snakemake/snakefmt#install) repository.
+
```

Comment:
You might consider adding python formatting style guide (aka `black`)

Reply:
added on : https://github.com/WayScience/CytoPipe/pull/5/commits/7564f7a4b657c4dd35ce6e0d8705d305845f0926